src/partials/_footer.hbs

```diff
@@ -0,0 +1,19 @@
+<footer class="footer">
+	<div class="footer__container">
+		<div class="footer__links">
+			<a href="#" class="footer__link">Privacy Policy</a>
+			<a href="#" class="footer__link">Terms &amp; Conditions</a>
+			<p class="footer__text">A Modern Eden &copy; 2016</p>
+		</div>
+
+		<div class="footer__site__logo">
```

Comment:
This particular element of your component is ambiguous to me. You should never need to use double underscores more than once in a class name. With a BEM naming convention of `[block]__[element]--[modifier]` I think you want something like `footer__site-logo` here where `site-logo` represents the `[element]` part of the name. Is that what you are going for here? Just remember that with BEM when you use multi-word names to use a single hyphen to separate words.